x/tokenfactory/keeper/commen_test.go

```diff
@@ -0,0 +1,77 @@
+package keeper_test
```

Comment:
This file should be named `common_test`, not `commen_test`. I believe there's an unwanted typo there.